### PR TITLE
fixes

### PR DESCRIPTION
**Pull request by Morshedk**

### CODE REVIEW

File: src/tsqlScheduler/Public/Set-DatabaseTask.ps1 [hunk 1]
```diff
@@ -11,14 +11,14 @@ Function Set-DatabaseTask {
     [Task] $Task
   )
   if ($PSCmdlet.ShouldProcess("Task $($Task.TaskUid) in $Database on $Server")) {
-
+$Task
```
Comment (taddison) [inline src/tsqlScheduler/Public/Set-DatabaseTask.ps1]: What does this do - is it for debugging?

**Review by taddison — CHANGES_REQUESTED** [review]
Can you either add a gitignore for the file ` .vs/slnx.sqlite ` or delete it locally if it's not used.  Doesn't need to be in the repo.

**Review by petervandivier — COMMENTED** [review]
prob wanna .gitignore the .sqlite file (you can also ignore local-only by adding it to `.git/info/exclude`

if you guys have any time/interest in adding a whole-module dependency to the project as well, `Invoke-DbaQuery` supports parameterised queries (to skip out on the `.Replace()` tickery :-p

**Comment by taddison** [comment]
Added the gitignore entry and removed the file.

Aiming to limit dependencies as long as possible, but that `replace` is pretty ugly.

**Review by taddison — APPROVED** [review]

**Comment by petervandivier** [comment]
fwiw - https://github.com/sqlcollaborative/Invoke-SqlCmd2 supports it as well and is obviously way lighter weight than dbatools proper

**Pull request merged**